foxpuppet/windows/browser/window.py

```diff
@@ -24,6 +25,15 @@ class BrowserWindow(BaseWindow):
     _tab_browser_locator = (By.ID, 'tabbrowser-tabs')
 
     @property
+    def is_tracking_shield_displayed(self):
```

Comment:
The window needs a `navBar` property, which is an instance of the `NavBar` class. Why don't you want to setup a hierarchical structure?